Sources/Expectation.swift

```diff
@@ -144,6 +144,17 @@ extension ExpectationType where ValueType == Bool {
   }
 }
 
+// Mark: Types
+
+extension ExpectationType {
+  public func beOfType(_ type: Any.Type) throws {
+    guard let value = try expression() else { throw failure("cannot determine type: value is nil") }
+    if Mirror(reflecting: value).subjectType != type {
+      throw failure("value is not type of '\(type)'")
```

Comment:
I think it would be nice to include the subject type too so the user can spot what the problem is easier.